native-cli/src/main/java/org/mule/weave/cli/pico/AbstractPicoExecCommand.java

```diff
@@ -0,0 +1,91 @@
+package org.mule.weave.cli.pico;
+
+import org.mule.weave.dwnative.cli.Console;
+import org.mule.weave.v2.utils.DataWeaveVersion;
+import picocli.CommandLine;
+import scala.Option;
+import scala.Predef;
+import scala.Tuple2;
+import scala.collection.JavaConverters;
+import scala.collection.immutable.Map;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.util.concurrent.Callable;
+
+public abstract class AbstractPicoExecCommand implements Callable<Integer> {
+    protected Console console;
+    @CommandLine.Option(names = {"--param", "-p"}, description = {"Parameter to be passed. All input parameters are accessible through the variable `params` of type object."})
+    protected java.util.Map<String, String> params = null;
+    @CommandLine.Option(names = {"--input", "-i"}, description = {"Declares a new input."}, paramLabel = "<Name=File>")
+    protected java.util.Map<String, File> inputs = null;
+    @CommandLine.Option(names = {"--verbose", "-v"}, description = {"Run the script as untrusted, which means that the script has no privileges."}, defaultValue = "false")
+    protected boolean verbose = false;
+    @CommandLine.Option(names = {"--untrusted"}, description = {"Run the script as untrusted, which means that the script has no privileges."}, defaultValue = "false")
+    protected boolean untrusted = false;
+    @CommandLine.Option(names = {"--language-level"}, description = {"The version of DW to be supported."})
+    protected String languageLevel = null;
+    @CommandLine.Option(
+            names = {"--privileges"},
+            description = {"A list of all allowed runtime privileges for this execution to have."},
+            split = ","
+    )
+    protected java.util.List<String> privileges = null;
+    @CommandLine.Option(names = {"--silent", "-s"}, description = {"Run in silent mode it reduces the logging on the standard output."})
```

Comment:
Should this be a `-q,--quiet` which is more commonly used for quite operations? For, in mvn commands.

Reply:
I  took the name from using `curl`. But it would be to put it up to a poll to see what people says. I will research a little bit more to see other cli commands.